packages/ilib-po/test/POFile.test.ts

```diff
@@ -37,17 +36,18 @@ describe("pofile", () => {
         expect.assertions(1);
 
         expect(() => {
-            const pof = new POFile({
-                pathName: undefined
-            });
+            const pof = new POFile(
+                // @ts-expect-error -- testing invalid args
+                { pathName: undefined }
+            );
         }).toThrow();
     });
 
     test("POFile constructor with basic options", () => {
         expect.assertions(2);
 
         const pof = new POFile({
-            pathName: "./testfiles/po/messages.po"
+            pathName: "./testfiles/po/messages.po",
```

Comment:
ugh, no trailing commas! I don't like that trend at all

Reply:
We can later agree on formatting rules together and add prettier if you'd like